alvr/dashboard/src/dashboard/components/setup_wizard.rs

```diff
@@ -28,7 +26,6 @@ fn index_to_page(index: usize) -> Page {
         1 => Page::ResetSettings,
         2 => Page::HardwareRequirements,
         3 => Page::SoftwareRequirements,
-        4 => Page::HandGestures,
```

Comment:
This doesn't modify the indices but they're modified in the `Page` enum

Reply:
whoops